handbook/engineering/commit_messages.md

```diff
@@ -0,0 +1,37 @@
+# Commit message guidelines
+
+Having consistent commit messages makes it easier to see at a glance what code has been affected by recent changes. It also makes it easier to search through our commit history.
+
+The body should contain
```

Comment:
Is this sentence left incomplete intentionally (to gather feedback)?

Reply:
Oops, my bad.